src/views/product-landing/server.ts

```diff
@@ -14,73 +14,79 @@ import { validateAgainstSchema } from 'lib/validate-against-schema'
 import { transformRawContentToProp, extractHeadings } from './helpers'
 import { ProductLandingViewProps } from './types'
 
-async function generateStaticProps({
-	product,
-	contentJsonFile,
-}: {
-	product: ProductData
-	contentJsonFile: string
-}): Promise<
-	ProductLandingViewProps & {
-		layoutProps: {
-			headings: TableOfContentsHeading[]
-			breadcrumbLinks: BreadcrumbLink[]
-			sidebarNavDataLevels: SidebarProps[]
-		}
-		product: ProductData
-	}
-> {
-	/**
-	 * Note: could consider other content sources. For now, JSON.
-	 * Asana task: https://app.asana.com/0/1100423001970639/1201631159784193/f
-	 */
-	const jsonFilePath = path.join(process.cwd(), contentJsonFile)
-	const CONTENT = JSON.parse(fs.readFileSync(jsonFilePath, 'utf8'))
+const generateGetStaticProps = (product: ProductData) => {
+	const getStaticProps = async (): Promise<
+		GetStaticPropsResult<
+			ProductLandingViewProps & {
+				layoutProps: {
+					headings: TableOfContentsHeading[]
+					breadcrumbLinks: BreadcrumbLink[]
+					sidebarNavDataLevels: SidebarProps[]
+				}
+				product: ProductData
+			}
+		>
+	> => {
+		/**
+		 * Note: could consider other content sources. For now, JSON.
+		 * Asana task: https://app.asana.com/0/1100423001970639/1201631159784193/f
+		 */
+		const jsonFilePath = path.join(
+			process.cwd(),
+			`src/content/${product.slug}/product-landing.json`
+		)
+		const CONTENT = JSON.parse(fs.readFileSync(jsonFilePath, 'utf8'))
```

Comment:
🗒️ We were passing a file path string in anyways, so I updated this to dynamically generate that string from the product slug. If we change the location or naming of these files in the future, we only have to update this string in one place rather than many places.